### PR TITLE
10 add clases bodkia

### PR DESCRIPTION
**Pull request by bodkia22**

### CODE REVIEW

File: src/YPS.Infrastructure/Services/ClassService.cs [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+            _context.Classes.Add(newClass);
+            await _context.SaveChangesAsync();
+
+            return await _context.Classes.SingleOrDefaultAsync(x => x.Id == newClass.Id);
+        }
+
+        private bool NumberValidation(long number) => number > 0 && number <= 12;
+
+        private bool CharacterValidation(string character) => character.Length == 1/* && Regex.IsMatch(character, "[A-Za-zÀ-ÿ]",RegexOptions.IgnoreCase)*/;
+
```
Comment (DmMov) [inline src/YPS.Infrastructure/Services/ClassService.cs]: Get rid of this comment

File: src/YPS.Application/SchoolRequests/Commands/CreateSchoolRequest/CreateSchoolRequestCommand.cs [hunk 2]
```diff
@@ -14,5 +14,6 @@ public sealed class CreateSchoolRequestCommand : IRequest<long>
         public string Email { get; set; }
         public string PhoneNumb { get; set; }
         public bool Confirmation { get; set; }
+
     }
```
Comment (DmMov) [inline src/YPS.Application/SchoolRequests/Commands/CreateSchoolRequest/CreateSchoolRequestCommand.cs]: Useless empty line

File: src/YPS.Application/Pupils/Commands/CreatePupil/CreatePupilCommand.cs [hunk 3]
```diff
@@ -44,7 +45,7 @@ public async Task<CreateUserResponse> Handle(CreatePupilCommand request, Cancell
                 if (res.Failures == null || !res.Failures.Any())
                 { 
                     string password = _randomGenerator.RandomPassword();
-                    User createdUser = await _userService.CreateUser(request.User, password, 1, request.SchoolId);
+                    User createdUser = await _userService.CreateUser(request.User, password, 1, 1);
```
Comment (DmMov) [inline src/YPS.Application/Pupils/Commands/CreatePupil/CreatePupilCommand.cs]: Why did you do this?

File: src/YPS.Application/Headmasters/Commands/CreateHeadmaster/CreateHeadMasterCommand.cs [hunk 4]
```diff
@@ -15,13 +15,13 @@
...
     {
         public UserPartial User { get; set; }
         public string Password { get; set; }
         public string Link { get; set; }
 
-        public class CreateHeadMasterCommandHandler : IRequestHandler<CreateHeadMasterCommand, CreateUserResponse>
+        public class CreateHeadMasterCommandHandler : IRequestHandler<CreateHeadMasterCommand, CreatedResponse>
         {
 
             private readonly IUserService _userService;
```
Comment (DmMov) [inline src/YPS.Application/Headmasters/Commands/CreateHeadmaster/CreateHeadMasterCommand.cs]: The 26th line is unnecessary

File: src/YPS.Application/Pupils/Commands/CreatePupil/CreatePupilCommand.cs [hunk 5]
```diff
@@ -12,13 +12,14 @@
...
-    public sealed class CreatePupilCommand : IRequest<CreateUserResponse>
+    public sealed class CreatePupilCommand : IRequest<CreatedResponse>
     {
         public UserPartial User { get; set; }
         public long ClassId { get; set; }
         public long SchoolId { get; set; }
 
-        public class CreatePupilCommandHandler : IRequestHandler<CreatePupilCommand, CreateUserResponse>
+
+        public class CreatePupilCommandHandler : IRequestHandler<CreatePupilCommand, CreatedResponse>
```
Comment (DmMov) [inline src/YPS.Application/Pupils/Commands/CreatePupil/CreatePupilCommand.cs]: Remove the 21th line

File: src/YPS.WebUI/Controllers/HeadmastersController.cs [hunk 6]
```diff
@@ -5,15 +5,15 @@
 using System.ComponentModel.DataAnnotations;
 using System.Linq;
 using System.Threading.Tasks;
-using YPS.Application.Auth.Command.CreateHeadMaster;
+using YPS.Application.Headmasters.Commands.CreateHeadmaster;
 using YPS.Application.HeadMasters.Queries.CheckMasterRegisterLink;
```
Comment (DmMov) [inline src/YPS.WebUI/Controllers/HeadmastersController.cs]: Capitalize the word master. It was capitalized, but you changed it changing the other parts of the namespase

Comment (DmMov) [reply src/YPS.WebUI/Controllers/HeadmastersController.cs]: Dont forget to change it everywhere

**Review by DmMov — APPROVED** [review]

**Review by shagi9 — APPROVED** [review]

**Review by Karonuk — APPROVED** [review]

**Pull request merged**